test/k8sT/manifests/fqdn-httpd.yaml

```diff
@@ -37,3 +37,14 @@ spec:
                 values:
                 - k8s1
                 - k8s2
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fqdn-httpd-svc
+spec:
+  ports:
+  - port: 80
+  selector:
+   app: fqdn-httpd
```

Comment:
I'm not sure you picky is yaml but there's a missing space here.

Reply:
I am! I would have noticed this later and it would bug me forever! thanks